timezone/src/modules/Timezone.rb

```diff
@@ -245,20 +251,25 @@ def get_zonemap
     # @return	the number of the region that contains the timezone
     #
     def Set(zone, really)
-      zmap = get_zonemap
+      # Do not update the timezone if it's forced and it was already set
+      if (Mode.installation || Mode.update) && readonly && !@timezone.empty?
```

Comment:
I think that we should allow to change the timezone in firstboot. The PBI refers to inst-sys and the installed system, but the user is allowed to change the timezone of a running system. I know that we could consider firstboot as part of the installation, but IMHO there's no point in asking for timezone during firstboot if you don't want the user to change it.